server/repository/migrations/postgres/2022-04-27T13-00_create_document_table/up.sql

```diff
@@ -17,7 +17,8 @@ CREATE TABLE document
     status document_status NOT NULL,
     comment TEXT,
     owner_name_id TEXT REFERENCES name (id),
-    context TEXT
+    context TEXT,
+    is_sync_update BOOLEAN NOT NULL DEFAULT FALSE
```

Comment:
Moved this straight here since it's not in production yet and the alter statement fails to recognise that the column exists even though it does...